### DISCUSSION TITLE
Castellar font removed for drop caps?

### DISCUSSION THREAD

**Discussion by Cityyankee24** [neanes/neanes#792] [General]
There was a font for drop caps I preferred called 'Castellar', has it been removed? I do not see it as a drop down selection now on page settings.

Thanks!

**Comment by basil** [comment]
Is there a particular release where this was last working, and a particular release when it started failing?

**Comment by danielgarthur** [comment]
That font has never been bundled with the app. Is it still installed on your computer?

**Comment by Cityyankee24** [comment]
Looks like I have the same version as my PC (posting from my laptop, both Windows). Below some screenshots. Are fonts pulled from somewhere else on the machine? Thanks for the help!!

![Version](https://github.com/neanes/neanes/assets/129484822/7e95e97f-09e4-47cf-9389-2de3f0260c6e)
![Castellar_Font_Sample](https://github.com/neanes/neanes/assets/129484822/e1055018-4337-4a8c-b2ad-7cb5b847b7d3)
![Castellar_Font](https://github.com/neanes/neanes/assets/129484822/eb1a2eb9-8b79-45c4-bbfb-b5d0b42175b5)

**Comment by danielgarthur** [comment]
To make sure I understand, are you saying it works on one Windows machine, but not on another? If so, are they both the same version of Windows? Which version of Windows?

If the font is installed on your computer, it should appear in the list of fonts. If you just installed it, you will have to close and re-open Neanes, since Neanes won't detect fonts that are installed while Neanes is running.

To check if the font is installed, open `Settings` in Windows, then search for `Font settings`.

<img width="403" alt="image" src="https://github.com/neanes/neanes/assets/4132779/ebded200-8aad-4bb2-b007-bbf1f4a5c46c">

You can also check to see whether it appears in the font list in a word processor like Microsoft Word or WordPad.

**Reply by Cityyankee24** [reply]
Same Windows version on Windows 11, but I did not realized the fonts were sourced from the local machine. Castellar is not an installed font on my desktop and it is on my laptop. I think that would explain it not showing.

Edit: I was able to find and install the font on my PC. Works fine. Thanks for your assistance!
